tools/generators/release-package/README.md

```diff
@@ -0,0 +1,45 @@
+# release-package
+
+Workspace Generator which automates release process stages for @fluentui (v9) package.
```

Comment:
might want to add documentation that it's only for initial preview and stable releases, it doesn't need to be run at any other time

Reply:
sounds good , also what about the name ? https://github.com/microsoft/fluentui/pull/28505#discussion_r1263643891

Reply:
added complete release process graph, should be good now